test/apiv2/80-kube.at

```diff
@@ -79,3 +79,71 @@ t DELETE libpod/play/kube $YAML 200 \
 rm -rf $TMPD
 
 # vim: filetype=sh
```

Comment:
I am not a vim user but I think this comment should stay at the bottom of the file.

Reply:
👍